l10n_th_withholding_tax_cert_form/data/mail_template.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" ?>
+<odoo >
+
+        <!--Email template -->
+        <record id="withholding_tax_cert_form_email_template" model="mail.template">
+            <field name="name">Withholding Tax Cert : Send by email</field>
+            <field name="model_id" ref="l10n_th_withholding_tax_cert.model_withholding_tax_cert"/>
+            <field name="subject">${object.supplier_taxid}</field>
+            <field name="partner_to">${object.partner_ids}</field>
+            <field name="body_html" type="xml">
+                <div style="margin: 0px; padding: 0px;">
+                    <p style="margin: 0px; padding: 0px; font-size: 13px;">
+                        Dear ${object.supplier_partner_id.name}.I'm Attach withholdding tax pdf report with email.Please check the validity of the file.<br/>
```

Comment:
```suggestion
                        Dear ${object.supplier_partner_id.name}.
                        <br /><br />
                        Here is your <strong>Withholding Tax Certificate</strong>. Please check the validity of the attachment.
                        <br /><br />
                        Do not hesitate to contact us if you have any questions.<br/><br/>
```

Reply:
I suggest changing the mail-template.

Reply:
Fixed.
@flook-chaiwat , Thanks for your suggestion.